add warns

sojalei opened the pull request:
Fixes #8

sojalei committed:
add warns

sojalei commented:
need refactoring, i'm lazy

evgfilim1 commented on lang/warns.json:
```diff
@@ -0,0 +1,26 @@
+  "en": {
+    "help": "This module can warn a person\nAvailable commands:\n/max_warns <num> - maximum amount warn\n/warn - reply to the person\n/unwarn - reply to the person",
+    "max_warns" : "Changed",
+    "last_warn" : "You cursed!",
```
"You have been cursed" would be right

evgfilim1 commented on lang/warns.json:
```diff
@@ -0,0 +1,26 @@
+{
+  "en": {
+    "help": "This module can warn a person\nAvailable commands:\n/max_warns <num> - maximum amount warn\n/warn - reply to the person\n/unwarn - reply to the person",
```
I think you should fix it, write what `/warn` and `/unwarn` means (also on other languages)

sojalei replied:
/warn means warn?

evgfilim1 replied:
You can write smth like this: `Reply to somebody's message with /warn to warn him/her`

evgfilim1 commented on lang/warns.json:
```diff
@@ -0,0 +1,26 @@
+    "last_warn" : "You cursed!",
+    "help.max_warns" : "Usage `/max_warns <num>`",
+    "reply_to_bot" : "It's me!",
+    "not_admin" : "You not admin"
```
"You are not admin" would be right

evgfilim1 commented on lang/warns.json:
```diff
@@ -0,0 +1,26 @@
+    "not_admin" : "You not admin"
+  },
+  "ru": {
+    "help": "Модуль умеющий предупреждать\nДоступный команды:\n/max_warns <цифра> - максимальное количество предупреждений\n/warn - ответьте человеку\n/unwarn - ответьте человеку",
```
* Grammar. Should be `Модуль, умеющий`
* Mistype. Should be  `Доступные команды`

evgfilim1 commented on modules/warns.py:
```diff
@@ -0,0 +1,101 @@
+from bases import BaseTelegramModule
+import pickle
```
You should fix some issues in this code. I've explained you in Telegram

sojalei committed:
Fixed errorss

sojalei committed:
Update warns.json

sojalei committed:
checks small warnings

sojalei committed:
Merge remote-tracking branch 'origin/master'

evgfilim1 commented on config/main.json.example:
```diff
@@ -1,10 +0,0 @@
-{
```
Why have you deleted this file?

sojalei committed:
fix problem with save max_warns, remove a negative number in /unwarn

sojalei committed:
edit long lines

sojalei committed:
bad codacy, bad...

evgfilim1 reviewed (changes requested):
Please, return `config/main.json.example`, it is deleted in your fork

sojalei committed:
add example

evgfilim1 committed:
Update main.json.example

evgfilim1 commented:
I'll fix it a bit, then merge

evgfilim1 committed:
Some fixes

Updated lang/warns.json
Added forgotten "kicked" key
Code cleanups and small bug fixes

evgfilim1 committed:
Removed unneeded `help` overload

evgfilim1 reviewed (changes requested):
Need bugfixes. I've explained you in Telegram.

sojalei committed:
fixes bugs
